### ISSUE TITLE
Add 'Lazy' or 'Easy' mode

### ISSUE THREAD

**Issue by alonando** [entertrained/entertrained-issues#7]
I almost don't want to request this feature because it defeats the purpose of practicing the "full" keyboard a little bit but since you have reading-mode already then this could be something in between.

Have a 'lazy' or 'easy' mode that will allow the user to only type the alphabet characters, any other character will be skipped automatically, so no punctuation, quoting, symbols or capitalization are needed to type correctly. For white space (spacing between words), user should be able to type 'space', 'enter' or 'tab' multiple times without affecting the start of the next word or paragraph.

Basically the intention for this mode is that the user only types lowercase letters and numbers and any blank space character between words or 'tokens'.

Most typing speed test videos I see online are of typing just lowercase words without any punctuation, symbols, uppercase characters etc. This would enable using Entertrained in this way.

Finally 'stats' should be kept separate if you are on 'easy' mode, since your 'easy' wpm is different than the full wmp.

**Comment by caderek** [comment]
Hey,
sorry for the late answer - I read it, and didn't know what to answer at the moment, wanting to came back to it later, and finally forgot about it.

I'm still conflicted about this feature, one one hand I can see some scenarios where it can be useful - like retyping notes or textbooks to better memorize the material (after I add custom texts upload), but for general typing practice I see it as detrimental. It also considerably complicates the code, because as you said - treating it on equal terms with proper practice wouldn't be ideal, and it would require some separate stats/scoring.

I'm leaving this issue open with low priority for now, if others have some ideas or opinions about this feature, please let me know below.

**Comment by fbsb** [comment]
The main usecase I see for this (which btw is also the main reason for me using entertrained and others apps) is learning a new keyboard layout.

In the beginning one might want to keep it simple, by skipping numbers, symbols and ignore capitalization and enable each category as one progresses and becomes more comfortable. 

This could be done with a sort of a level option and storing stats for each level.

Anyway thanks for the awesome work! Keep it up!

**Comment by localcatgirl-dev** [comment]
Another example is when you type in other language that have a LOT of diacritics and is so much harder to write it casually because the diacritics is not optimized for qwerty and other keyboard layout.

Czech language for example: Příliš žluťoučký kůň úpěl ďábelské ódy.
